drivers/hmis/app/controllers/hmis/graphql_controller.rb

```diff
@@ -13,14 +13,31 @@ class GraphqlController < Hmis::BaseController
 
     def execute
       variables = prepare_variables(params[:variables])
-      query = params[:query]
-      operation_name = params[:operationName]
       context = {
         current_user: current_hmis_user,
       }
       case params[:schema]
       when :hmis
-        result = HmisSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+        if params[:_json]
+          # We have a batch of operations
+          queries = params[:_json].map do |param|
+            {
+              query: param[:query],
+              operation_name: param[:operationName],
+              variables: variables,
```

Comment:
Testing it out now, this actually doesn't work for me when making requests with the batch link. It needs to be
```suggestion
     variables: prepare_variables(param[:variables]),
```

Reply:
Should be done now